registry-config.yaml

```diff
@@ -0,0 +1,23 @@
+# https://docs.docker.com/registry/configuration/
+
+version: 0.1
+log:
+  level: debug # info, warn, error
+  fields:
+    service: registry
+
+storage:
+  cache:
+    blobdescriptor: inmemory
```

Comment:
I think this is why we lost the tag after I restarted the server (i.e., it was cached in memory)

According to https://docs.docker.com/registry/configuration/#cache, we probably need to put it in Redis.  We can do that in a follow-up, but it needs to be filed.

Reply:
Should we create that Issue in our Telescope repo or this repo?

Reply:
Maybe here